tests/unit/modules/test_zcbuildout.py

```diff
@@ -243,8 +243,16 @@ def test_get_bootstrap_content(self):
             "",
             buildout._get_bootstrap_content(os.path.join(self.tdir, "var", "tb", "1")),
         )
+
+        if (
+            salt.utils.platform.is_windows()
+            and os.environ.get("GITHUB_ACTIONS_PIPELINE", "0") == "0"
```

Comment:
Why are we checking for `GITHUB_ACTIONS_PIPELINE` here. Wouldn't we always want linebreaks to be `\r\n` on Windows?

Reply:
Under GH pipelines, we use `ssh` to run command on the VM, a windows build of `OpenSSH`.
Line breaks on windows, through `ssh` are `\n`...